modules/olm-disconnected-operatorhub-disconnected-mode.adoc

```diff
@@ -0,0 +1,289 @@
+
+.. For the CSV file creation, find the following lines in the `bundle.yaml` file:
++
+[source,yaml]
+----
+data:
+  clusterServiceVersions: |
+----
++
+Omit those lines, but save a new file consisting of the full CSV resource
+content beginning with the following lines, removing the prepended `-`
+character:
++
+[source,yaml]
+.Example `clusterserviceversion.yaml` file snippet
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+[...]
+----
+
+.. For the CRD file creation, find the following line in the `bundle.yaml` file:
++
+[source,yaml]
+----
+  customResourceDefinitions: |
+----
++
+Omit this line, but save new files consisting of each, full CRD resource content
+beginning with the following lines, removing the prepended `-` character:
++
+[source,yaml]
+.Example `customresourcedefinition.yaml` file snippet
+----
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+[...]
+----
++
+.. For the package file creation, find the following line in the `bundle.yaml`
+file:
++
+[source,yaml]
+----
+  packages: |
+----
++
+Omit this line, but save a new file consisting of the package content beginning
+with the following lines, removing the prepended `-` character, and ending with
+a `packageName` entry:
++
+[source,yaml]
+.Example `package.yaml` file
+----
+channels:
+- currentCSV: etcdoperator.v0.9.4
+  name: singlenamespace-alpha
+- currentCSV: etcdoperator.v0.9.4-clusterwide
+  name: clusterwide-alpha
+defaultChannel: singlenamespace-alpha
+packageName: etcd
+----
+
+. **Create an Operator catalog image.**
+
+.. Save the following to a Dockerfile, for example named
+`custom-registry.Dockerfile`:
++
+[source,go]
+----
+FROM registry.redhat.io/openshift4/ose-operator-registry:v4.2 AS builder
+
+COPY manifests manifests
+
+RUN /bin/initializer -o ./bundles.db
+
+FROM scratch
+
+COPY --from=builder /build/bundles.db /bundles.db
+COPY --from=builder /build/bin/registry-server /registry-server
+COPY --from=builder /bin/grpc_health_probe /bin/grpc_health_probe
+
+EXPOSE 50051
+
+ENTRYPOINT ["/registry-server"]
+
+CMD ["--database", "bundles.db"]
+----
+
+.. Use the `podman` command to create and tag the container image from the
+Dockerfile:
++
+----
+$ podman build -f custom-registry.Dockerfile \
+    -t <internal_registry_route>/<namespace>/custom-registry <1>
+----
+<1> Tag the image for the internal registry of the disconnected {product-title}
+cluster and any namespace.
+
+. **Push the Operator catalog image to a registry.**
++
+Your new Operator catalog image must be pushed to a registry that the
+{product-title} can access. This can be the internal registry of the cluster
+itself or another registry that the disconnected cluster has network access to,
+such as an on-premise Quay Enterprise registry.
++
+For this example, login and push the image to the internal registry of the
+disconnected {product-title} cluster
++
+----
+$ podman push <internal_registry_route>/<namespace>/custom-registry
```

Comment:
Should we specify what namespace to push this too? 
What authentication needs to be provided to podman for this push to work?

Reply:
@ecordell @shawn-hurley My understanding was it could be any namespace. Should we state one? Existing or have them create a new one?

Reply:
This is the same registry and info that is in the main disconnected doc correct? we should just follow that IMO.